gusb/gusb-context.c

```diff
@@ -126,12 +129,18 @@ g_usb_context_dispose (GObject *object)
 		g_source_remove (priv->hotplug_poll_id);
 		priv->hotplug_poll_id = 0;
 	}
+	if (priv->idle_events_id > 0) {
```

Comment:
Suggestion:
```
g_clear_handle_id (&priv->idle_events_id, g_source_remove);
```

Reply:
Wow, ancient, but fair enough!

Reply:
Ubuntu LTS for Google :/

Reply:
Ui, annoying. I guess this means meson is only 0.48?

 * [bionic (18.04LTS)](https://packages.ubuntu.com/bionic/meson): 0.45.1-2ubuntu0.18.04.2
 * [bionic-updates](https://packages.ubuntu.com/bionic-updates/meson): 0.45.1-2ubuntu0.18.04.2
 * [focal (20.04LTS)](https://packages.ubuntu.com/focal/meson): 0.53.2-2ubuntu2

I guess I will not be able to use a disabler for adding the umockdev tests ... oh well

Reply:
It's only Focal and newer we care about IIRC. But yes, the meson dep has to be low too :/

For the tests, are they the same exact tests you have in libfprint? If so, I don't think we need to duplicate them tbh.